internal/k8s/entity_test.go

```diff
@@ -21,3 +21,20 @@ func TestImmutableFilter(t *testing.T) {
 		t.Errorf("Expected Job entity, actual: %+v", jobs)
 	}
 }
+
+func TestLoadBalancers(t *testing.T) {
+	entities, err := ParseYAMLFromString(BlorgBackendYAML)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	lbs := ToLoadBalancers(entities)
+	if len(lbs) != 1 {
+		t.Fatalf("Expected 1 loadbalancer, actual: %d", len(lbs))
```

Comment:
maybe `"Expected 1 loadbalancer, got %d: %v", len(lbs), lbs` for debugging purposes?

Reply:
done